packages/seacas/libraries/ioss/src/generated/Iogn_GeneratedMesh.C

```diff
@@ -1550,7 +1550,7 @@ namespace Iogn {
     }
 
 #if 1
-    std::memcpy(rotmat, res, 9 * sizeof(double));
+    std::memcpy(rotmat.data(), res, 9 * sizeof(double));
```

Comment:
It is safe.  `res` is double[3][3].

I will remove the memcpy; convert res to std::array similar to rotmat.  However, current code is safe.  std::array memory layout guarantees it will work correctly.